pandas/io/json/_json.py

```diff
@@ -717,7 +717,7 @@ def read_json(
 "data":[["a","b"],["c","d"]]\
 }}\
 '
-    >>> pd.read_json(StringIO(_), orient='split')
+    >>> pd.read_json(StringIO(_), orient='split')  # doctest: +SKIP
```

Comment:
Since we are already creating the dataframe and calling `to_json` I think it'd be better to save the dataframe to a `StringIO` object, display it so users can continue to see how the json looks like, and then make this work by reading the `StringIO` object. What do you think?